lib/v2/scau/yjsy.js

```diff
@@ -0,0 +1,27 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const link = 'https://yjsy.scau.edu.cn/208/list.htm';
+    const response = await got(link);
+    const $ = cheerio.load(response.data);
+    const list = $('#wp_news_w25 tr');
+
+    ctx.state.data = {
+        title: '华南农业大学研究生院',
+        link,
+        description: '通知公告',
+        item:
+            list &&
+            list.toArray().map((item) => {
+                item = $(item);
+                const a = item.find('a');
+                return {
+                    title: a.text(),
```

Comment:
The selector includes everything as you can see from the first item of https://github.com/DIYgod/RSSHub/pull/13455#issuecomment-1744310879

Reply:
Thank your comment. I have found my mistake. And I have corrected it.